app/models/follow_manager.rb

```diff
@@ -0,0 +1,36 @@
+class FollowManager
+  def self.create(user, url)
+    UserFollow.create(user_id: user.id,
+                     followable_id: followable_id(url),
+                     followable_type: type(url))
+  end
+  
+  def self.destroy(user, url)
+    UserFollow.where(user_id: user.id)
+              .where(followable_id: followable_id(url))
+              .destroy_all
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 10) spaces for indenting an expression spanning multiple lines.